teachbooks/external_content/config.py

```diff
@@ -39,6 +39,15 @@ def get_teachbooks_config(config_file: Path) -> dict[str, str]:
                 )
                 raise ValueError(msg)
             cfg["attribution_location"] = loc
+
+    # Extract language from Sphinx configuration
+    if (
+        "sphinx" in config
+        and "config" in config["sphinx"]
+        and "language" in config["sphinx"]["config"]
+    ):
+        cfg["language"] = config["sphinx"]["config"]["language"]
```

Comment:
```suggestion
cfg["language"] = config.get("sphinx", {}).get("config", {}).get("language", "en")
```
This should work in a single line :smile:

Reply:
I had a single line, and than ruff started complaining

Reply:
Would you suggestion be ruff-approved?

Reply:
Yeah. Ruff didn't like the nested if statements. 

My suggestion tries to get a value from a dict, and if it's missing it returns an empty dict (so the next `.get` call doesn't fail). Ruff won't complain about it.